base_comment_template/views/base_comment_template_view.xml

```diff
@@ -39,19 +39,19 @@
                 <sheet>
                     <div class="oe_button_box" name="button_box">
                         <widget
-                            name="web_ribbon"
-                            title="Archived"
-                            bg_color="bg-danger"
-                            attrs="{'invisible': [('active', '=', True)]}"
-                        />
+              name="web_ribbon"
```

Comment:
These changes shouldn't be here.

Reply:
@pedrobaeza they came along while running pre-commit, any way to prevent only part of the code?

Reply:
In which local folder are you handling the migration? May it contain spaces in it? Try to put it again as it was, and perform pre-commit from a folder without spaces in its path to check.

Reply:
@pedrobaeza I will try cloning the repo again.